### ISSUE TITLE
Deploy without provisioning

### ISSUE THREAD

**Issue by ip1981** [NixOS/nixops#607]
Afte ec2 infrastructure is created, I want just to deploy stuff. Unfortunately nixops bails out if not provided with aws credentials. By the way, Hetzner backend supports this workflow.

Nixops has some options for gradual deploy: create, create and copy, create, copy and switch. I'd like to have an option to skip the create step.

**Comment by Nadrieril** [comment]
If I understand your problem correctly, you could use the `none` backend, which lets you deploy via ssh to any pre-existing machine (see https://nixos.org/nixops/manual/#opt-deployment.targetEnv).

**Comment by ip1981** [comment]
Yes, I could. But ec2 backend is good in provisioning :) I just want to be sure my Jenkins won't change any resources. Automatic provisioning is good until it gets some information that is not known beforehand, like IP addresses. IOW if it were fully declarative.

I'm thinking about a small (hopefully) patch to nixops, or use other tools for provisioning.
